Return-Path: <julien@cristau.org>
X-Original-To: maks@dual
Delivered-To: maks@dual
Received: from dual (localhost.localdomain [127.0.0.1])
	by dual (Postfix) with ESMTP id 6E49C240C6
	for <maks@dual>; Sat, 15 May 2010 21:45:52 +0200 (CEST)
X-Original-To: maks@stro.at
Delivered-To: maks@stro.at
Received: from baikonur.stro.at [213.239.196.228]
	by dual with POP3 (fetchmail-6.3.16)
	for <maks@dual> (single-drop); Sat, 15 May 2010 21:45:52 +0200 (CEST)
Received: from master.debian.org (master.debian.org [70.103.162.29])
	by baikonur.stro.at (Postfix) with ESMTP id 430795C00A
	for <maks@stro.at>; Sat, 15 May 2010 21:30:11 +0200 (CEST)
Received: from coloquinte.cristau.org ([91.121.16.100])
	by master.debian.org with esmtps (TLS1.0:DHE_RSA_AES_256_CBC_SHA1:32)
	(Exim 4.69)
	(envelope-from <julien@cristau.org>)
	id 1ODNBm-0004tk-80
	for maks@stro.at; Sat, 15 May 2010 19:38:22 +0000
Received: from radis.liafa.jussieu.fr (unknown [IPv6:2a01:e35:139e:7980:221:6aff:fe12:9638])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by coloquinte.cristau.org (Postfix) with ESMTPSA id 32F94BBA0
	for <maks@debian.org>; Sat, 15 May 2010 21:38:19 +0200 (CEST)
Received: from julien by radis.liafa.jussieu.fr with local (Exim 4.71)
	(envelope-from <julien@radis.liafa.jussieu.fr>)
	id 1ODNBi-0000CW-Da
	for maks@debian.org; Sat, 15 May 2010 21:38:18 +0200
Resent-From: Julien Cristau <julien@radis.liafa.jussieu.fr>
Resent-Date: Sat, 15 May 2010 21:38:18 +0200
Resent-Message-ID: <20100515193818.GY7781@radis.liafa.jussieu.fr>
Resent-To: maks@debian.org
X-Original-To: julien@cristau.org
Received: from coloquinte.cristau.org (localhost.localdomain [127.0.0.1])
	by coloquinte.cristau.org (Postfix) with ESMTP id 7BD26BBD5
	for <julien@cristau.org>; Thu, 13 May 2010 20:56:14 +0200 (CEST)
Received: from toccata.ens-lyon.org (toccata.ens-lyon.org [140.77.166.68])
	(using TLSv1 with cipher ADH-AES256-SHA (256/256 bits))
	(No client certificate requested)
	by coloquinte.cristau.org (Postfix) with ESMTPS id AEC83BADD
	for <julien@cristau.org>; Thu, 13 May 2010 20:56:08 +0200 (CEST)
X-Virus-Scanned: Debian amavisd-new at toccata.ens-lyon.org
Received: from toccata.ens-lyon.org ([127.0.0.1])
	by localhost (toccata.ens-lyon.org [127.0.0.1]) (amavisd-new, port 10024)
	with ESMTP id ZXs+ATNCvAZE for <julien@cristau.org>;
	Thu, 13 May 2010 20:55:56 +0200 (CEST)
Received: from gabe.freedesktop.org (gabe.freedesktop.org [131.252.210.177])
	by toccata.ens-lyon.org (Postfix) with ESMTP id 23FE2840B6
	for <julien.cristau@ens-lyon.org>; Thu, 13 May 2010 20:55:46 +0200 (CEST)
Received: from gabe.freedesktop.org (localhost [127.0.0.1])
	by gabe.freedesktop.org (Postfix) with ESMTP id 5FE809E84B
	for <julien.cristau@ens-lyon.org>; Thu, 13 May 2010 11:55:45 -0700 (PDT)
X-Original-To: dri-devel@lists.freedesktop.org
Received: from mx1.redhat.com (mx1.redhat.com [209.132.183.28])
	by gabe.freedesktop.org (Postfix) with ESMTP id 5BF2C9E773
	for <dri-devel@lists.freedesktop.org>;
	Thu, 13 May 2010 11:55:30 -0700 (PDT)
Received: from int-mx08.intmail.prod.int.phx2.redhat.com
	(int-mx08.intmail.prod.int.phx2.redhat.com [10.5.11.21])
	by mx1.redhat.com (8.13.8/8.13.8) with ESMTP id o4DItTL9031581
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=OK)
	for <dri-devel@lists.freedesktop.org>; Thu, 13 May 2010 14:55:30 -0400
Received: from localhost.localdomain (ihatethathostname.lab.bos.redhat.com
	[10.16.43.238])
	by int-mx08.intmail.prod.int.phx2.redhat.com (8.13.8/8.13.8) with ESMTP
	id o4DItSJc014462
	for <dri-devel@lists.freedesktop.org>; Thu, 13 May 2010 14:55:28 -0400
From: Adam Jackson <ajax@redhat.com>
To: dri-devel@lists.freedesktop.org
Subject: [PATCH] drm/edid: Fix 1024x768@85Hz
Date: Thu, 13 May 2010 14:55:28 -0400
Message-Id: <1273776928-16924-1-git-send-email-ajax@redhat.com>
X-Scanned-By: MIMEDefang 2.67 on 10.5.11.21
X-BeenThere: dri-devel@lists.freedesktop.org
X-Mailman-Version: 2.1.11
Precedence: list
List-Id: Direct Rendering Infrastructure - Development
	<dri-devel.lists.freedesktop.org>
List-Unsubscribe: <http://lists.freedesktop.org/mailman/options/dri-devel>,
	<mailto:dri-devel-request@lists.freedesktop.org?subject=unsubscribe>
List-Archive: <http://lists.freedesktop.org/archives/dri-devel>
List-Post: <mailto:dri-devel@lists.freedesktop.org>
List-Help: <mailto:dri-devel-request@lists.freedesktop.org?subject=help>
List-Subscribe: <http://lists.freedesktop.org/mailman/listinfo/dri-devel>,
	<mailto:dri-devel-request@lists.freedesktop.org?subject=subscribe>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: dri-devel-bounces+julien.cristau=ens-lyon.org@lists.freedesktop.org
Errors-To: dri-devel-bounces+julien.cristau=ens-lyon.org@lists.freedesktop.org
X-Virus-Scanned: ClamAV using ClamSMTP
Content-Length: 1025
Lines: 28

Having hsync both start and end on pixel 1072 ain't gonna work very
well.  Matches the X server's list.

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 drivers/gpu/drm/drm_edid.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/drm_edid.c b/drivers/gpu/drm/drm_edid.c
index 18f41d7..10348d3 100644
--- a/drivers/gpu/drm/drm_edid.c
+++ b/drivers/gpu/drm/drm_edid.c
@@ -335,7 +335,7 @@ static struct drm_display_mode drm_dmt_modes[] = {
 		   DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC) },
 	/* 1024x768@85Hz */
 	{ DRM_MODE("1024x768", DRM_MODE_TYPE_DRIVER, 94500, 1024, 1072,
-		   1072, 1376, 0, 768, 769, 772, 808, 0,
+		   1168, 1376, 0, 768, 769, 772, 808, 0,
 		   DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC) },
 	/* 1152x864@75Hz */
 	{ DRM_MODE("1152x864", DRM_MODE_TYPE_DRIVER, 108000, 1152, 1216,
-- 
1.7.0.1

_______________________________________________
dri-devel mailing list
dri-devel@lists.freedesktop.org
http://lists.freedesktop.org/mailman/listinfo/dri-devel
